src/libostree/ostree-repo-static-delta-core.c

```diff
@@ -873,3 +873,176 @@ _ostree_repo_static_delta_dump (OstreeRepo                    *self,
 
   return TRUE;
 }
+
+/**
+ * ostree_repo_static_delta_is_signed:
+ * @self: Repo
+ * @delta_id: delta path
+ * @error: Error
+ *
+ * Verify if the superblock of static-data has signature(s).
+ *
+ * Returns: TRUE if static data superblock has signatures, FALSE otherwise.
+ *
+ * Since: 2019.6
+ */
+gboolean
+ostree_repo_static_delta_is_signed (OstreeRepo       *self,
+                                    const char       *delta_id,
```

Comment:
IMHO: have sense to change API of this function -- allow to pass additional OUT variable STRV which will be filled with names of signature types found. If passed NULL, then no need to return names.
rationale: since this function could be used by external soft -- it would help to decide if it have to fail upgrade or allow it in case if the superblock is signed with unknown/unsupported signature type.
@fdanis-oss @em- what do you think?

Reply:
good idea, thanks